src/Components/RoomForm/RoomForm.js

```diff
@@ -77,9 +76,6 @@ function RoomForm({getCircleCode}) {
           onChange={(e) => { setRoomname(e.target.value) }}
           />
         </section>
-        <section className='input-container'>
-          <label>Group Code:</label> <h2>{ roomID }</h2>
-        </section>
```

Comment:
just going with group name now?

Reply:
Yeah as we discussed with Ryan he doesn't want to start with the group code from the get-go